sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientTest.java

```diff
@@ -32,35 +36,34 @@
     @Override
     protected void beforeTest() {
         beforeTestSetup();
-        if (interceptorManager.isPlaybackMode()) {
-            client = clientSetup(pipeline -> {
-                this.pipeline = pipeline;
-                return new KeyClientBuilder()
-                    .pipeline(pipeline)
-                    .vaultUrl(getEndpoint())
-                    .buildClient();
-            });
-        } else {
-            client = clientSetup(pipeline -> {
-                this.pipeline = pipeline;
-                return new KeyClientBuilder()
-                    .pipeline(pipeline)
-                    .vaultUrl(getEndpoint())
-                    .buildClient();
-            });
-        }
     }
 
-    @Test
-    public void encryptDecryptRsa() throws Exception {
+    private void initializeKeyClient(HttpClient httpClient) {
+        pipeline = getHttpPipeline(httpClient, KeyServiceVersion.getLatest());
```

Comment:
KeyServiceVersion should also be parameterized

Reply:
Explained above. 

The test class is to test over CryptographyServiceClient instead of KeyClient.
There are some other test classes which tests KeyClient, and we have parameterized test on KeyServiceVersion over there.